doc/CHANGES.rst

```diff
@@ -19,8 +19,8 @@ Security related fixes
 Features
 ++++++++
 
-- Add wildcard rewrite to sub host name in virtualHostMonster.
-  (`#317 <https://github.com/zopefoundation/Zope/issues/317>`_)
+- Add support of IPv6 named hosts in VirtualHostMonster
+  (`#395 <https://github.com/zopefoundation/Zope/issues/395>`_)
```

Comment:
Oh wait, the link is also wrong, even if github will redirect the #395 PR, there was no issue opened ( except the old one on launchpad https://bugs.launchpad.net/zope2/+bug/699865  but I guess we don't need to reference it here )

Reply:
I just amended this commit.

As I said on https://github.com/zopefoundation/Zope/pull/400 , it's unclear if this should be listed in Bugfixes or new features (but let's discuss this in https://github.com/zopefoundation/Zope/pull/400 )